integration/smartbft/smartbft_test.go

```diff
@@ -1643,7 +1643,8 @@ var _ = Describe("EndToEnd Smart BFT configuration test", func() {
 			// If backoff's default settings are left, it will take 20-25 seconds after the 7th attempt to reach the 8th attempt.
 			// If the backoff is made controllable, as here in the test, the maximum time between attempts will be 3 seconds.
 			for i := 0; i < 7; i++ {
-				Eventually(ordererRunners[0].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("pickfirstBalancer: UpdateSubConnState.*IDLE connection error"))
+				// look for grpc Info message
+				Eventually(ordererRunners[0].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("Received SubConn state update.*IDLE connection error"))
```

Comment:
IDLE ConnectionError

Reply:
Thank you. Relying on these log messages is fragile!

Reply:
Totally agree with you. 
But I can't think of any other way to show and prove that there are problems with an unconfigured backoff grpc.